browsermob-core/src/main/java/net/lightbody/bmp/proxy/selenium/SeleniumProxyHandler.java

```diff
@@ -634,6 +636,12 @@ protected void wireUpSslWithCyberVilliansCA(String host, SslRelay listener) {
           }
       }
 
+      public void cleanSslWithCyberVilliansCA(){
+          if(deleteDirectoryTask != null) {
+              new Thread(deleteDirectoryTask).start();
```

Comment:
You can just call deleteDirectoryTask.run() here, no need to spin up a new Thread.